en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+not owned by the creator. If necessary, the system administrator can
+delete such accounts to manage resources better.
+
+!!! Note
+    This feature is only available via the Self Registration and Account confirmation REST APis. 
+    Currently, WSO2 IS does not support this feature via the End User Dashboard.
+
+The following sections walk you through configuring and trying out
+self-registration via SMS and Email Channels.
+    
+??? Warning "If you have migrated from a previous IS version"
+    If you have migrated from a previous IS version, ensure that the `IdentityMgtEventListener` with the
+    `         orderId=50        ` is set to **false** and that the Identity
+    Listeners with `         orderId=95        ` and
+    `         orderId=97        ` are set to **true** in the
+    `         <IS_HOME>/repository/conf/deployment.toml        ` file.
+   
+    !!! Note 
+        If there is no such entries for `event.default_listener.xxx`, in `deployment.toml`, 
+        no need to configure the following.
+    
+    
+    ``` java
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
+    
+## Configuring self-registration
+
+Follow the steps given below to register users for the super tenant,
+which is `         carbon.super        `.
+
+1.  Add the following properties to the `deployment.toml` file in the `IS_HOME/repository/conf` folder to 
+configure the the identity server to sent confirmation emails.
+    
+    !!! Note
+        You need to add this configuration only if you wish to configure WSO2 IS to send confirmation 
+        emails. Alternatively, you can use your own email managing mechanism.
+
+    ``` toml
+    [output_adapter.email]
+    from_address= "wso2iamtest@gmail.com"
+    username= "wso2iamtest"
+    password= "Wso2@iam70"
+    hostname= "smtp.gmail.com"
+    port= 587
+    enable_start_tls= true
+    enable_authentication= true
+    ```    
+    
+2. Add the following properties to the `deployemnt.toml` to configure notification handling and 
+self registration to support multiple notification channels.
+
+    ```toml
+    [identity_mgt.notification]
+    default_notification_channel = "EMAIL"
+    enable_resolve_notification_channel = true
+    
+    [identity_mgt.user_self_registration]
+    SkipAccountLockOnVerifiedPreferredChannel = false
+    enable_detailed_api_response = true
+    ```    
+    
+    <table>
+    <tr>
+        <th>Property Name</th>
+        <th>Functionality</th>
+    </tr>
+    <tr>
+    <td>default_notification_channel</td>
+    <td>
+        <ul>
+            <li>Default notification channel. If the user has not specified a notification channel
+            user will be notified with the defined channel.</li>
```

Comment:
with --> via the defined channel

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549